LEGO1/lego/legoomni/src/build/legocarbuildpresenter.cpp

```diff
@@ -47,8 +53,8 @@ LegoCarBuildAnimPresenter::~LegoCarBuildAnimPresenter()
 	m_unk0xc8.GetRoot()->SetNumChildren(0);
 	*m_unk0xc8.GetRoot()->GetChildren() = NULL;
 
-	if (m_unk0x14c) {
-		delete m_unk0x14c;
+	if (m_mainSourceId) {
+		delete m_mainSourceId;
```

Comment:
should be `delete[]`

Reply:
Huh, I learned something today. It does not affect the binary code at all since `LegoChar` does not have a destructor, but it's definitely cleaner that way.